[2025-02-28T14:49:20.666+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-28T14:49:20.723+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl.data_quality_check scheduled__2025-02-28T06:00:00+00:00 [queued]>
[2025-02-28T14:49:20.747+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl.data_quality_check scheduled__2025-02-28T06:00:00+00:00 [queued]>
[2025-02-28T14:49:20.750+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-02-28T14:49:20.788+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): data_quality_check> on 2025-02-28 06:00:00+00:00
[2025-02-28T14:49:20.815+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=216) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-28T14:49:20.809+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_etl', 'data_quality_check', 'scheduled__2025-02-28T06:00:00+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl.py', '--cfg-path', '/tmp/tmp7ff0t4js']
[2025-02-28T14:49:20.823+0000] {standard_task_runner.py:91} INFO - Job 117: Subtask data_quality_check
[2025-02-28T14:49:20.822+0000] {standard_task_runner.py:63} INFO - Started process 218 to run task
[2025-02-28T14:49:20.976+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_etl.data_quality_check scheduled__2025-02-28T06:00:00+00:00 [running]> on host 083d33cb466d
[2025-02-28T14:49:21.411+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_etl' AIRFLOW_CTX_TASK_ID='data_quality_check' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-28T06:00:00+00:00'
[2025-02-28T14:49:21.413+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-28T14:49:21.471+0000] {python.py:237} INFO - Done. Returned value was: {'df_pass': {'city_id': {0: 1581130, 1: 1566083, 2: 1583992, 3: 1572151, 4: 1586203, 5: 1562798, 6: 1580240, 7: 1586896, 8: 1584071, 9: 1568574, 10: 1569684, 11: 1566166, 12: 1566319, 13: 1591449, 14: 1571058, 15: 1574023, 16: 1568510, 17: 1580410, 18: 1586443}, 'city_name': {0: 'Hanoi', 1: 'Ho Chi Minh', 2: 'Da Nang', 3: 'Nha Trang', 4: 'Can Tho', 5: 'Vinh', 6: 'Hue', 7: 'Buon Ma Thuot', 8: 'Da Lat', 9: 'Quy Nhon', 10: 'Pleiku', 11: 'Thanh Hoa', 12: 'Thai Nguyen', 13: 'Bac Ninh', 14: 'Phan Thiet', 15: 'My Tho', 16: 'Rach Gia', 17: 'Ha Long', 18: 'Ca Mau'}, 'country': {0: 'VN', 1: 'VN', 2: 'VN', 3: 'VN', 4: 'VN', 5: 'VN', 6: 'VN', 7: 'VN', 8: 'VN', 9: 'VN', 10: 'VN', 11: 'VN', 12: 'VN', 13: 'VN', 14: 'VN', 15: 'VN', 16: 'VN', 17: 'VN', 18: 'VN'}, 'collected_at': {0: '2025-02-28 14:48:44', 1: '2025-02-28 14:48:45', 2: '2025-02-28 14:48:45', 3: '2025-02-28 14:48:47', 4: '2025-02-28 14:48:48', 5: '2025-02-28 14:48:48', 6: '2025-02-28 14:48:51', 7: '2025-02-28 14:48:51', 8: '2025-02-28 14:48:51', 9: '2025-02-28 14:48:52', 10: '2025-02-28 14:48:52', 11: '2025-02-28 14:48:53', 12: '2025-02-28 14:48:53', 13: '2025-02-28 14:48:54', 14: '2025-02-28 14:48:54', 15: '2025-02-28 14:48:54', 16: '2025-02-28 14:48:55', 17: '2025-02-28 14:48:56', 18: '2025-02-28 14:48:56'}, 'temperature': {0: 20.4, 1: 26.44, 2: 22.47, 3: 22.93, 4: 25.04, 5: 20.32, 6: 20.57, 7: 20.9, 8: 16.39, 9: 23.52, 10: 19.37, 11: 19.65, 12: 19.9, 13: 20.03, 14: 23.43, 15: 24.9, 16: 27.67, 17: 19.13, 18: 25.57}, 'feels_like': {0: 20.82, 1: 26.44, 2: 23.17, 3: 23.63, 4: 25.56, 5: 20.86, 6: 21.22, 7: 21.29, 8: 16.59, 9: 24.23, 10: 19.74, 11: 20.2, 12: 20.3, 13: 20.49, 14: 24.15, 15: 25.51, 16: 29.59, 17: 19.63, 18: 26.14}, 'humidity': {0: 89, 1: 64, 2: 92, 3: 90, 4: 75, 5: 94, 6: 97, 7: 86, 8: 96, 9: 88, 10: 91, 11: 97, 12: 90, 13: 92, 14: 89, 15: 79, 16: 66, 17: 97, 18: 75}, 'pressure': {0: 1014, 1: 1012, 2: 1016, 3: 1016, 4: 1013, 5: 1014, 6: 1014, 7: 1014, 8: 1017, 9: 1016, 10: 1015, 11: 1015, 12: 1015, 13: 1015, 14: 1013, 15: 1013, 16: 1013, 17: 1016, 18: 1013}, 'wind_speed': {0: 4.75, 1: 6.42, 2: 4.36, 3: 2.09, 4: 5.04, 5: 1.71, 6: 1.58, 7: 3.04, 8: 1.13, 9: 2.45, 10: 2.87, 11: 1.37, 12: 1.99, 13: 3.9, 14: 5.0, 15: 4.41, 16: 4.73, 17: 1.29, 18: 2.27}, 'wind_direction': {0: 149, 1: 134, 2: 150, 3: 354, 4: 112, 5: 280, 6: 136, 7: 92, 8: 67, 9: 20, 10: 77, 11: 158, 12: 136, 13: 145, 14: 75, 15: 111, 16: 136, 17: 136, 18: 99}, 'cloud_cover': {0: 73, 1: 32, 2: 92, 3: 0, 4: 4, 5: 4, 6: 30, 7: 5, 8: 21, 9: 77, 10: 1, 11: 17, 12: 90, 13: 74, 14: 10, 15: 11, 16: 84, 17: 24, 18: 9}, 'weather': {0: 'broken clouds', 1: 'scattered clouds', 2: 'overcast clouds', 3: 'clear sky', 4: 'clear sky', 5: 'clear sky', 6: 'scattered clouds', 7: 'clear sky', 8: 'few clouds', 9: 'broken clouds', 10: 'clear sky', 11: 'few clouds', 12: 'overcast clouds', 13: 'broken clouds', 14: 'clear sky', 15: 'few clouds', 16: 'broken clouds', 17: 'few clouds', 18: 'clear sky'}, 'visibility': {0: 10000, 1: 10000, 2: 10000, 3: 10000, 4: 10000, 5: 10000, 6: 10000, 7: 10000, 8: 10000, 9: 10000, 10: 10000, 11: 10000, 12: 10000, 13: 10000, 14: 10000, 15: 10000, 16: 10000, 17: 10000, 18: 10000}, 'dew_point': {0: None, 1: None, 2: None, 3: None, 4: None, 5: None, 6: None, 7: None, 8: None, 9: None, 10: None, 11: None, 12: None, 13: None, 14: None, 15: None, 16: None, 17: None, 18: None}, 'precipitation': {0: 0.0, 1: 0.0, 2: 0.0, 3: 0.0, 4: 0.0, 5: 0.0, 6: 0.0, 7: 0.0, 8: 0.0, 9: 0.0, 10: 0.0, 11: 0.0, 12: 0.0, 13: 0.0, 14: 0.0, 15: 0.0, 16: 0.0, 17: 0.0, 18: 0.0}}, 'df_fail': {'city_id': {}, 'city_name': {}, 'country': {}, 'collected_at': {}, 'temperature': {}, 'feels_like': {}, 'humidity': {}, 'pressure': {}, 'wind_speed': {}, 'wind_direction': {}, 'cloud_cover': {}, 'weather': {}, 'visibility': {}, 'dew_point': {}, 'precipitation': {}}}
[2025-02-28T14:49:21.473+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-28T14:49:21.494+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_etl, task_id=data_quality_check, run_id=scheduled__2025-02-28T06:00:00+00:00, execution_date=20250228T060000, start_date=20250228T144920, end_date=20250228T144921
[2025-02-28T14:49:21.546+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-28T14:49:21.566+0000] {taskinstance.py:3503} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-02-28T14:49:21.569+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
